apps/dialtone-documentation/docs/components/mode-island.md

```diff
@@ -0,0 +1,834 @@
+---
+title: Mode Island
+description: Create independent sections with their own color modes.
+status: beta
+---
+
+<code-well-header>
+  <dt-stack gap="500" class="d-ai-flex-start d-w100p">
+    <dt-stack direction="row" gap="500" class="d-jc-space-between d-w100p">
+      <h4 class="d-headline--lg">Demo</h4>
+      <dt-dropdown
+        navigation-type="arrow-keys"
+        placement="bottom-end"
+      >
+        <template #anchor>
+          <dt-button
+            importance="outlined"
+            kind="muted"
+          >
+            <dt-stack gap="400" direction="row">
+              <span><strong>Mode:</strong> {{ currentMode.charAt(0).toUpperCase() + currentMode.slice(1) }}</span>
+              <span><strong>Contrast:</strong> {{ currentContrast.charAt(0).toUpperCase() + currentContrast.slice(1) }}</span>
+            </dt-stack>
+            <template #icon>
+              <dt-icon
+                size="300"
+                :name="currentModeIconName"
+              />
+            </template>
+          </dt-button>
+        </template>
+        <template #list>
+          <dt-list-item-group
+            heading-class="d-py4 d-px8 d-c-default d-fc-tertiary d-label--sm"
+            heading="Mode"
+          >
```

Comment:
I can see this changes the entire site configuration.
It could leave a unintended configuration behind.

Since this is a component to `create independent sections with their own color modes.` I will expect it will only change the below examples?




https://github.com/user-attachments/assets/f4b8bb8c-6b34-4d1d-af41-ca67b934bc00